cockpit/cats-plugin/cats.js

```diff
@@ -14,19 +14,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.camunda.bpm.cockpit.plugin.failedjobs.resources;
 
-import javax.ws.rs.Path;
-//import javax.ws.rs.PathParam;
+export default {
+  id: "Cats Plugin",
+  pluginPoint: "cockpit.dashboard",
+  priority: 5,
+  render: container => {
+    const header = document.createElement("h1");
+    header.innerHTML = "<h1>Cats!</h1>";
 
-import org.camunda.bpm.cockpit.plugin.failedjobs.FailedJobsPlugin;
-import org.camunda.bpm.cockpit.plugin.resource.AbstractPluginRootResource;
+    const mainContent = document.createElement("div");
+    mainContent.innerHTML = "<img src=\"http://thecatapi.com/api/images/get?size=medium\" width=\"400\" />";
 
-@Path("plugin/" + FailedJobsPlugin.ID)
-public class FailedJobsPluginRootResource extends AbstractPluginRootResource {
-
-  public FailedJobsPluginRootResource() {
-    super(FailedJobsPlugin.ID);
+    container.appendChild(header);
+    container.appendChild(mainContent);
```

Comment:
Alternatively, we can remove this and replace it with
```
container.innerHTML = `
<h1>Cats!</h1>
<img src="http://thecatapi.com/api/images/get?size=medium" width="400" />
`;
```

Reply:
👍